src/buildMathML.js

```diff
@@ -639,6 +639,9 @@ const buildExpression = function(expression, options) {
         const group = expression[i];
         groups.push(buildGroup(group, options));
     }
+
+    // TODO(kevinb): combine \\not with mrels and mords
```

Comment:
MathJax combines `\\not` with certain relation operators and outputs the combined HTML entity inside an `<mo>` and for mords, it puts the HTML entity for `not` after the identifier inside an `<mi>`.